### DISCUSSION TITLE
[572410]Steel Division: Normandy 44 + 3DLC

### DISCUSSION THREAD

**Discussion by TodRichter** [pjy612/SteamManifestCache#143] [Share]
appid:572410
keys
```
572411;19017ed9ddb4c15f7803607f33f247a1c2816ea9c19d9994f4a46c1c95d5fa33
572412;321459ca7df971af5f121f32857c16223eb03585ab93bc6bac8839bf544864f3
639063;6c6e7ff33cbc13a2cc7b1af926591bef4a4cdc18ce8aeb39cfdc937080f65fb0
639064;aa8ae552357350197a26fca0e30fbb6a3cd86880d5c261cdef8583011c6e6b85
639065;06f6ed8dd66c68e41eddadfd0a916be5a9cabcdc00891be0b364934c540ef8b1
639066;f68b84d33f5c2135077d7f15878b1fe294d967fb8908d2bc14800fb031828e6e
639067;3bd33135a3b8451bd929d4b1cfe06c76beacb4fc263b3fa8dff49d02a37235ca
639060;ba5457c2ed67d0419f1bc24c14a8fa74f26f0f7cc44a71aae1ece9268dc298cd
```
CODE:FA0D847F0E11A47C
E-mail:a1179875218@163.com

**Comment by pjy612** [comment]
DLC 密钥不全
https://steamdb.info/app/572410/depots/

**Reply by TodRichter** [reply]
228986;51dd3611d28621644730736f3bb1fd6b960053a45cd79123f2b9a80c9181dad5
228990;44d8c45ce229a11c4f231a3d2a350eaf80b0d69a8af938ec7ccca720f694b0e8
572411;19017ed9ddb4c15f7803607f33f247a1c2816ea9c19d9994f4a46c1c95d5fa33
572412;321459ca7df971af5f121f32857c16223eb03585ab93bc6bac8839bf544864f3
639063;6c6e7ff33cbc13a2cc7b1af926591bef4a4cdc18ce8aeb39cfdc937080f65fb0
639064;aa8ae552357350197a26fca0e30fbb6a3cd86880d5c261cdef8583011c6e6b85
639065;06f6ed8dd66c68e41eddadfd0a916be5a9cabcdc00891be0b364934c540ef8b1
639066;f68b84d33f5c2135077d7f15878b1fe294d967fb8908d2bc14800fb031828e6e
639067;3bd33135a3b8451bd929d4b1cfe06c76beacb4fc263b3fa8dff49d02a37235ca
639060;ba5457c2ed67d0419f1bc24c14a8fa74f26f0f7cc44a71aae1ece9268dc298cd

**Reply by pjy612** [reply]
其实你可以编辑 自己的帖子的

**Reply by pjy612** [reply]
查收

**Comment by TodRichter** [comment]
[depotcache.zip](https://github.com/pjy612/SteamManifestCache/files/13772289/depotcache.zip)
